Update DTEx212_single_particle_tracking.ipynb

JChonpca opened the pull request:
1. Update DTEx212_single_particle_tracking.ipynb to Torch Version
2. Update the formatting in DTEx211
3. Adding DTEx212 links to README.md

Commit:
Update DTEx212_single_particle_tracking.ipynb

Commit:
Update READNE-pypi.md

Commit:
Update README.md

Commit:
Update DTEx212_single_particle_tracking.ipynb

correct typos, polish formats

Pwhsky commented:
Hi Jiacheng, the tutorial looks good and polished. 
A couple of small things are missing when it comes to formatting, otherwise it looks complete to me.

Put "." after each numbering of the chapters, so that it becomes:
### 2.1. chapter name
instead of  
### 2.1 chapter name

* If possible, remove some duplicate import statements or duplicate `%matplotlib` commands.
* Chapter  **5.1.** should have 3 #'s instead of 2 for appropriate size.
* Keep the character limit to 78 per line, so a couple of places needs a line break.

Commit:
Update DTEx212_single_particle_tracking.ipynb

1. remove duplicate import and %matplotlib
2. PEP check
3. Missing # in ###5.1.1

JChonpca commented:
Hi Alex, thanks for your reviews. I have updated one more version for the things you mentioned.

1. Put "." after each numbering of the chapters: it was already done in the Commit [[84fc607](https://github.com/DeepTrackAI/DeepTrack2/pull/346/commits/84fc607afc7e018cb70aa4239d11285cdb121add)]
2. Removing some duplicate import statements and `%matplotlib` command.
3. Adding missing `#` in `##5.1.1`
4. PEP-8 check using the online Formatter.

Pwhsky reviewed (commented):
Looks complete to me, well done.

Commit:
MNIST+README

Commit:
Polish
